### DISCUSSION TITLE
MorphTo field label for `morph_id` missing when `morph_type` hasn't been selected yet

### DISCUSSION THREAD

**Discussion by mrleblanc101** [laravel/nova-issues#4924] [Ideas & Feature Requests]
- Laravel Version: 9.34.0
- Nova Version: 4.15.2
- PHP Version: 8.1
- Database Driver & Version: mariadb 10.4
- Operating System and Version: macOS 12.6
- Browser type and version: Chrome 105

### Description:
When using a MorphTo field, the morh_id (second field) has no label until the morph_type (first field) is selected.
![Capture d’écran, le 2022-10-14 à 17 16 08](https://user-images.githubusercontent.com/17392251/195945643-4070c76c-9796-425c-9324-09fb839f49c7.png)
Once you select the morph_type, the label appear.
![Capture d’écran, le 2022-10-14 à 17 16 22](https://user-images.githubusercontent.com/17392251/195945814-bd5c40a0-040f-4c7a-a184-a6b2795b2654.png)

This looks odd, the whole second field should be completely hidden and not just disabled or there should be a placeholder label.

**Comment by mrleblanc101** [comment]
This is only a visual glitch as everything work as expected.

**Comment by stephan-v** [comment]
Agreed, this is confusing as hell when you encounter this. It might even be better to hide the field altogether and show it after a MorphTo selection has been made.
